src/pages/advance/clerk-idp.mdx

```diff
@@ -0,0 +1,69 @@
+import { Callout } from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+
+# Use Clerk as an OAuth 2.0 Identity Provider 
+
+Learn how to use Clerk as an Identity provider to facilitate Single Sign-On (SSO) with other applications that support the OAuth2 protocol. 
+
+<Images
+  width={496}
+  height={564}
+  src="/images/advance/clerk-idp-flow.svg"
+  alt="Clerk SSO IDP Flow"
+/>
+
+
+## Step 1: Retrieve SSO Provider Callback URL
+
+From your SSO provider, retrieve the **Callback URL**. This is where your users will be redirected to after a successful authentication and is required to successfully create your OAuth application in the next step.
+
+
+## Step 2: Create an OAuth Application
+
+Next, you need to create an OAuth Application in Clerk via the [Backend API](https://clerk.com/docs/reference/backend-api/tag/OAuth-Applications).
+
+```sh copy filename="POST /v1/oauth_applications"
+curl 
+ -X POST https://api.clerk.com/v1/oauth_applications \
+ -H "Authorization: Bearer <CLERK_SECRET_KEY>"  \
+ -H "Content-Type: application/json" \
+ -D {"callback_url":"https://oauth-client.com/oauth2/callback", "name": "oauth_app_1"}
```

Comment:
It might be worth it to point out that the callback URL needs to be retrieved from the application/OAuth client (Circle.so for example), and name can be anything you want to help identify which OAuth application this is.